packages/xo-server/src/api/mirror-backup.mjs

```diff
@@ -0,0 +1,146 @@
+const SCHEMA_SETTINGS = {
+  type: 'object',
+  properties: {
+    '*': {
+      type: 'object',
+      properties: {
+        concurrency: {
+          type: 'number',
+          minimum: 0,
+          optional: true,
+        },
+        maxExportRate: {
+          type: 'number',
+          minimum: 0,
+          optional: true,
+        },
+      },
+      additionalProperties: true,
+    },
+  },
+  optional: true,
+}
+
+export function createJob({ schedules, ...job }) {
+  job.userId = this.apiContext.user.id
+  job.type = 'mirrorBackup'
+  return this.createBackupNgJob(job, schedules).then(({ id }) => id)
+}
+
+createJob.permission = 'admin'
+createJob.params = {
+  mode: {
+    enum: ['full', 'delta'],
```

Comment:
Isn't it better to use `incremental` instead of `delta`, to fit with new backup naming?

Reply:
delta is still the type of existing logs and jobs . Either we acept and handles the 3 values or we implement a migration path

Reply:
Yes, let's avoid changing the internals for now.